components/profile_popover/profile_popover.jsx

```diff
@@ -345,10 +372,23 @@ class ProfilePopover extends React.Component {
             />
         );
 
+        const supportsChannelAdmin = this.props.isLicensed;
+        const isChannelAdmin = supportsChannelAdmin && (Utils.isChannelAdmin(this.props.isLicensed, this.props.channelMember.roles) || this.props.channelMember.scheme_admin);
+
+        let roleTitle;
+        if (Utils.isSystemAdmin(this.props.user.roles)) {
+            roleTitle = <span className='user-popover__role'>{'System Admin'}</span>;
+        } else if (Utils.isInRole(this.props.teamMember.roles, 'team_admin')) {
+            roleTitle = <span className='user-popover__role'>{'Team Admin'}</span>;
+        } else if (isChannelAdmin && !this.props.isSearch) {
+            roleTitle = <span className='user-popover__role'>{'Channel Admin'}</span>;
```

Comment:
I think these strings should be i18n rather than hard coded in english?

Reply:
Yes, absolutely. Fixed.